package NOptimizedModel;

message TKeyValue {
    optional string Key = 1;
    optional string Value = 2;
}

message TNamedFloatVector {
    optional string Key = 1;
    repeated float Values = 2;
}

message TInputFrame {
    repeated TKeyValue Headers = 1;
    repeated TNamedFloatVector ExternalInputs = 2;

    optional string WebL3ModelsFlagsVersion = 3;
    repeated TKeyValue WebL3ModelsVersionsRewrites = 4;
}

message TInputDump {
    repeated TInputFrame Frames = 1;
}
